fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -1900,7 +1900,7 @@ public void finishTransactionNew(TransactionState transactionState, Set<Long> pu
         // do after transaction finish
         GlobalStateMgr.getCurrentState().getOperationListenerBus().onStreamJobTransactionFinish(transactionState);
         GlobalStateMgr.getCurrentState().getLocalMetastore().handleMVRepair(transactionState);
-        LOG.info("finish transaction {} successfully", transactionState);
+        LOG.debug("finish transaction {} successfully", transactionState);
```

Comment:
I think this log needs to keep.

Reply:
> I think this log needs to keep.

only ignore this log in checkpoint thread right now, PTAL again.